django_mfa/views.py

```diff
@@ -14,8 +14,13 @@
 from django.contrib.auth import load_backend
 from django.contrib import auth, messages
 from django.urls import reverse, reverse_lazy
-from django.utils.http import is_safe_url
-from django.utils.translation import ugettext as _
+try:
+    from django.utils.http import url_has_allowed_host_and_scheme as is_safe_url
+except ImportError:
+    # Django < 4.0 fallback
+    from django.utils.http import is_safe_url
+
+from django.utils.translation import gettext as _
```

Comment:
did you mean to change ugettext to gettext?

Reply:
Yes. Second bullet point here: https://docs.djangoproject.com/en/4.0/releases/3.0/#miscellaneous-1